getClients.js

```diff
@@ -0,0 +1,22 @@
+exports.getAll = function(event, context, callback) {
+  const connection = db.connect(event.stageVariables);
+  var query = 'SELECT * FROM Clients ORDER BY' + event.queryStringParameters.sort';
+
```

Comment:
String is malformed here. Needs 

var query = 'SELECT * FROM Clients ORDER BY' + 'event.queryStringParameters.sort';